services/heyArnoldApi.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs').promises;
+const http = require('https');
+
+Promise.all([
+  get(`https://hey-arnold-api.herokuapp.com/api/v1/characters/random?count=${count}`),
+  get(`https://hey-arnold-api.herokuapp.com/api/v1/gifs/random?count=${count}`)
+])
+  .then(([{ results: characters }, { results: gifs }]) => {
+    return {
+      characters: characters.map(({ id, name, image }) => ({ id, name, image })),
+      gifs: gifs.map(({ id, gifLink }) => ({ id, gifLink }))
+    };
+  })
+  .then(obj => fs.writeFile('./content.json', JSON.stringify(obj)));
```

Comment:
you should use your results from your API directly in your code. You don't need to write the results to a file.